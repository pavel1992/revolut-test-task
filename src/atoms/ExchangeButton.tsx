import styled from 'styled-components';

import { colors } from '../styleConstants/colors';

export const ExchangeButton = styled.button`
  padding: 14px 50px;
  background-color: ${colors.pink};
  color: ${colors.white};
  border-radius: 24px;
  border: none;
  cursor: pointer;
  &:focus {
      outline: none;
  }

  &:hover {
    background-color: ${colors.hoveredPink};
  }

  &:disabled {
    cursor: not-allowed;
    background-color: ${colors.disabledGray}
  }
`;
