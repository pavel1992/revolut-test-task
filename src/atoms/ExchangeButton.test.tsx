import * as React from 'react';
import renderer from 'react-test-renderer';

import { ExchangeButton } from './ExchangeButton';


test('renders ExchangeButton correctly', () => {
  const render = renderer
    .create(<ExchangeButton>111</ExchangeButton>)
    .toJSON();
  expect(render).toMatchSnapshot();
});
