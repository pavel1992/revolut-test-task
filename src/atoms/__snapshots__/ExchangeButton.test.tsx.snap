// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ExchangeButton correctly 1`] = `
<button
  className="sc-AxjAm hBDLuT"
>
  111
</button>
`;
